packages/zmq-android.4.0-8/findlib

```diff
@@ -0,0 +1 @@
+zmq
```

Comment:
Hello,

without such findlib files for _zmq-android_ and _stdint_, ocamlfind can't find this packages when I compile my code.